aom/src/main/java/com/nedap/archie/rminfo/MetaModels.java

```diff
@@ -135,6 +134,26 @@ public void selectModel(String rmPublisher, String rmPackage, String rmRelease)
 
     }
 
+    private AomProfile getAomProfileWithSchemaId(BmmModel selectedBmmModel) {
+        if(selectedBmmModel != null) {
+            for (AomProfile profile : aomProfiles.getProfiles()) {
+                if (profile.getRmSchemaPattern().stream().anyMatch(pat -> selectedBmmModel.getSchemaId().matches(pat))) {
+                    return this.selectedAomProfile = profile;
```

Comment:
You already set `this.selectedAomProfile` in the calling method.
```suggestion
                    return profile;
```

Reply:
oops, that was left over from earlier code. thanks